test/Microsoft.DocAsCode.MarkdigEngine.Tests/RowsTest.cs

```diff
@@ -0,0 +1,172 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.MarkdigEngine.Tests
+{
+    using Xunit;
+
+    public class RowTest
+    {
+        private void TestMarkup(string source, string expected)
+        {
+            var marked = TestUtility.MarkupWithoutSourceInfo(source, "Topic.md");
+            Assert.Equal(expected.Replace("\r\n", "\n"), marked.Html);
+        }
+
+        [Fact]
+        [Trait("Related", "Row")]
+        public void RowTest_SimpleRow()
+        {
+            var source = @":::row:::
+    :::column:::
+        This is where your content goes.
+    :::column-end:::
+:::row-end:::
+";
+            var expected = @"<section class=""row"">
+<div class=""column"">
+<p>This is where your content goes.</p>
+</div>
+</section>
+";
+            TestMarkup(source, expected);
+        }
+
+        [Fact]
+        [Trait("Related", "Row")]
+        public void RowTest_WithColumns()
+        {
+            var source = @":::row:::
+    :::column:::
+        This is where your content goes.
+    :::column-end:::
+:::row-end:::
+";
+            var expected = @"<section class=""row"">
+<div class=""column"">
+<p>This is where your content goes.</p>
+</div>
+</section>
+";
+            TestMarkup(source, expected);
+        }
+
+        [Fact]
+        [Trait("Related", "Row")]
+        public void RowTest_WithColumnsWithSpans()
+        {
+            var source = @":::row:::
+    :::column span=""2"":::
+        This is where your content goes.
+    :::column-end:::
+    :::column:::
+        This is where your content goes.
+    :::column-end:::
+:::row-end:::
+";
+            var expected = @"<section class=""row"">
+<div class=""column span2"">
```

Comment:
Howabout leveraging the naming convention applied to [Grid in Bootstrap](http://getbootstrap.com/docs/4.0/layout/grid/) to use `col-2` instead of `span2`?

Reply:
Oh docs, we already have a competing convention that uses `col`, but it's a very different implementation with different uses. Additionally, the mooncake header already untilizes Bootstrap, so that would be a definite conflict.